client/src/components/ProductLaunchPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {useTheme} from '@material-ui/core'
+
+const SvgProductLaunchLogo = props => {
```

Comment:
I might be misunderstanding here, but why is this file ProductLaunchPage if it's the logo component?

Reply:
Oh shoot I did mean for it to be logo I guess when I renamed the files I wasn't paying as much attention I'll fix that before I merge thanks